### ISSUE TITLE
Missing "accessibilityTitle" in scale-tab-header 

### ISSUE THREAD

**Issue by wgmaik** [telekom/scale#1422]
**Scale Version**
3.0.0-beta.117

**Framework and version**
Angular 14

**Current Behavior**
Missing possibility to set "Alt" text for screen readers on scale-tab-header.

**Expected Behavior**
Possibility to set "accessibilityTitle" in scale-tab-header (to add additional information for screen readers).
For all "navigation" elements it should be possible to set an "accessibilityTitle"!!! Otherwise they are unusable/difficult to understand
for e.g. blind person that uses a SR. Maybe one could check all available components. 

**Desktop (please complete the following information):**

OS: Windows 10
Browser CHROME
Version 107.0.5304.107 (Offizieller Build) (64-Bit)

**Comment by acstll** [comment]
Hi @wgmaik!

The `accessibilityTitle` is a prop we use for components with `svg` content, namely icons and logos. In which case we add a [`<title>` tag inside the SVG](https://github.com/telekom/scale/blob/main/packages/components/src/components/icon/icon.tsx#L69).

Other components supporting attributes like `aria-label` in elements inside the shadow DOM may have a prop prefixed with `inner` to add these attributes to the right element. One example is [`innerAriaLabel` for the Button](https://github.com/telekom/scale/blob/main/packages/components/src/components/button/button.tsx#L65).

In the specific case of the Tab Header, the actual custom element `scale-tab-header` has the [`tab` role](https://w3c.github.io/aria/#tab). There is not inner element with semantics. So I would say you could add an `aria-label` directly to it, e.g.

```html
<scale-tab-header aria-label="My label">Different label or icon?</scale-tab-header>
```

Would this help solve the issue?

Thanks in advance and sorry for the delay in our reply!

**Comment by sulyok-daniel** [comment]
> Would this help solve the issue?

I think so, yes. Thanks.

**Comment by acstll** [comment]
Closing but please feel free to keep the conversation going if needed.
